packages/playwright-test/src/index.ts

```diff
@@ -29,6 +29,8 @@ export { addRunnerPlugin as _addRunnerPlugin } from './plugins';
 export const _baseTest: TestType<{}, {}> = rootTestType.test;
 export const store = _baseStore;
 
+addStackIgnoreFilter((frame: StackFrame) => frame.file.startsWith(path.join(__dirname, '..')));
```

Comment:
nit: require.resolve('../package.json') would be easier to reason about if this file is moved into a subfolder.